helm/crossplane/templates/crd-install/crd-np.yaml

```diff
@@ -1,4 +1,24 @@
 {{- if .Values.giantswarm.crds.install }}
+{{- if .Capabilities.APIVersions.Has "cilium.io/v2/CiliumNetworkPolicy" }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: {{ include "crdInstall" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  annotations:
+    # create hook dependencies in the right order
+    "helm.sh/hook-weight": "-7"
+    {{- include "crdInstallAnnotations" . | nindent 4 }}
+  labels:
+    app.kubernetes.io/component: {{ include "crdInstall" . | quote }}
+    {{- include "labels.selector" . | nindent 4 }}
+    role: {{ include "crdInstallSelector" . | quote }}
+spec:
+  egress:
+  - toEntities:
+    - kube-apiserver
+  endpointSelector: {}
+{{- else }}
```

Comment:
Don't you need to use the `---` separator here?

Reply:
it's one or the other (if-else), it never generates 2 objects

Reply:
ah ok, my bad, didn't notice that